11-17 17:21:28.370 3318-3318/org.tribler.android V/videoResult: Intent { dat=file:///storage/emulated/0/Movies/VID_2016-11-17_17_21_10.mp4 flg=0x1 } Bundle[null]
11-17 17:21:28.373 3318-3318/org.tribler.android V/MainActivity: onNewIntent: org.tribler.android.PUBLISH
11-17 17:21:28.373 3318-3318/org.tribler.android V/ACTION_PUBLISH: file:///storage/emulated/0/Movies/VID_2016-11-17_17_21_10.mp4
11-17 17:21:28.374 3318-3318/org.tribler.android V/showLoading: Creating torrent…
11-17 17:21:28.467 3318-3318/org.tribler.android W/IInputConnectionWrapper: showStatusIcon on inactive InputConnection
11-17 17:21:31.000 3318-3318/org.tribler.android V/createTorrent: /data/data/org.tribler.android/cache/VID_2016-11-17_17_21_10.mp4/VID_2016-11-17_17_21_10.mp4
11-17 17:21:31.337 3318-3318/org.tribler.android V/createTorrent: Torrent is now created!
11-17 17:21:31.340 3318-3318/org.tribler.android V/showLoading: Adding torrent…
11-17 17:21:31.717 3318-3404/org.tribler.android V/readLine: {"type": "torrent_discovered", "event": {"files": [["VID_2016-11-17_17_21_10.mp4", 37935392]], "name": "VID_2016-11-17_17_21_10.mp4", "trackers": [], "timestamp": 1479399691, "dispersy_cid": "c6ed6e0ac0d18e3ea6896d336c051963b9e9a95f", "infohash": "7a15d16c177cd161f5bee31ba66bb33a4d54cf23"}}
11-17 17:21:31.718 3318-3404/org.tribler.android V/onEvent: TorrentDiscoveredEvent; handlers: 1
11-17 17:21:31.724 3318-3318/org.tribler.android V/addTorrent: Torrent is now added!
11-17 17:21:31.742 3318-3318/org.tribler.android V/showLoading: null
11-17 17:21:48.586 3318-3404/org.tribler.android V/readLine: {"type": "tribler_exception", "event": {"text": "Unhandled Error\nTraceback (most recent call last):\n  File \"triblerd.py\", line 36, in <module>\n    Triblerd().run()\n  File \"triblerd.py\", line 31, in run\n    reactor.run()\n  File \"/data/data/org.tribler.android/files/lib/python2.7/site-packages/twisted/internet/base.py\", line 1194, in run\n    self.mainLoop()\n  File \"/data/data/org.tribler.android/files/lib/python2.7/site-packages/twisted/internet/base.py\", line 1203, in mainLoop\n    self.runUntilCurrent()\n--- <exception caught here> ---\n  File \"/data/data/org.tribler.android/files/lib/python2.7/site-packages/twisted/internet/base.py\", line 825, in runUntilCurrent\n    call.func(*call.args, **call.kw)\n  File \"/data/data/org.tribler.android/files/lib/python2.7/site-packages/Tribler/Core/TorrentChecker/torrent_checker.py\", line 122, in _task_select_tracker\n    session.connect_to_tracker().addCallbacks(*self.get_callbacks_for_session(session))\n  File \"/data/data/org.tribler.android/files/lib/python2.7/site-packages/Tribler/Core/TorrentChecker/session.py\", line 214, in connect_to_tracker\n    self.request = self.register_task(\"request\", agent.request('GET', bytes(url)))\nexceptions.UnicodeEncodeError: 'ascii' codec can't encode characters in position 234-237: ordinal not in range(128)\n"}}
11-17 17:21:48.586 3318-3404/org.tribler.android E/parseEvent: Unknown event type: tribler_exception
